apps/console/src/features/applications/components/settings/sign-on-methods/sign-on-methods.tsx

```diff
@@ -121,6 +122,7 @@ export const SignOnMethods: FunctionComponent<SignOnMethodsPropsInterface> = (
     const [ googleAuthenticators, setGoogleAuthenticators ] = useState<GenericAuthenticatorInterface[]>(undefined);
     const [ gitHubAuthenticators, setGitHubAuthenticators ] = useState<GenericAuthenticatorInterface[]>(undefined);
     const [ facebookAuthenticators, setFacebookAuthenticators ] = useState<GenericAuthenticatorInterface[]>(undefined);
+    // const [ microsoftAuthenticators, setMicrosoftAuthenticators ] = useState<GenericAuthenticatorInterface[]>(undefined);
```

Comment:
Let'' not commit any commented code. Add them by checking the git history later if needed.

Reply:
resolved